### PR TITLE
office-js, office-js-preview, new PowerPoint APIs

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Please fill in this template.

- [ x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x ] Provide a URL to documentation or source code which provides context for the suggested changes: [shapes](https://docs.microsoft.com/en-us/office/dev/add-ins/powerpoint/shapes)
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

ONLY A RELATIVELY SMALL NUMBER OF APIs HAVE BEEN ADDED. MOST OF THE DIFF IS THE SIDE EFFECT OF MOVING APIs FROM ONE PLACE TO ANOTHER WITHIN THE FILES.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -111566,553 +111729,576 @@ declare namespace PowerPoint {
...
-         *
-         * @param propertyNames A comma-delimited string or an array of strings that specify the properties to load.
          */
-        load(propertyNames?: string | string[]): PowerPoint.TextRange;
+        group = "Group",
         /**
-         * Queues up a command to load the specified properties of the object. You must call `context.sync()` before reading the properties.
-         *
-         * @param propertyNamesAndPaths `propertyNamesAndPaths.select` is a comma-delimited string that specifies the properties to load, and `propertyNamesAndPaths.expand` is a comma-delimited string that specifies the navigation properties to load.
+         * The shape is a line
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:111776]:
```suggestion
         * The shape is a line.
```

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -113118,126 +113519,336 @@ declare namespace PowerPoint {
...
+        interface SlideScopedCollectionData {
+            items?: PowerPoint.Interfaces.SlideData[];
+        }
+        /** An interface describing the data returned by calling `slideMasterCollection.toJSON()`. */
+        interface SlideMasterCollectionData {
+            items?: PowerPoint.Interfaces.SlideMasterData[];
+        }
+        /**
+         * @remarks
+         * [Api set: PowerPointApi 1.0]
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:113540]: Just curious: Was there a 1.0?

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Spot check looks good

File: types/office-js-preview/index.d.ts (hunk 2 above)
Comment (Rick-Kirkham) [reply types/office-js-preview/index.d.ts:113540]: Good catch. Fixed.

**Pull request merged**